backoffice/backoffice/hep/migrations/0011_alter_hepworkflow_status.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.6 on 2025-11-27 12:22
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("hep", "0010_alter_hepdecision_action"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="hepworkflow",
+            name="status",
+            field=models.CharField(
+                choices=[
+                    ("approval", "Waiting for approval"),
+                    ("core_selection_approval", "Waiting for CORE selection approval"),
+                    ("merge_approval", "Waiting for merging conflicts"),
+                    ("fuzzy_matching", "Waiting for matching approval"),
+                    ("multiple_exact_matches", "Multiple exact matches"),
```

Comment:
would that be an error? `error_multiple_exact_matches`

Reply:
yes, will change it!